packages/edit-site/src/components/page-patterns/grid-item.js

```diff
@@ -265,66 +196,8 @@ function GridItem( { categoryId, item, ...props } ) {
 						) }
 					</Flex>
 				</HStack>
-				<DropdownMenu
-					icon={ moreVertical }
-					label={ __( 'Actions' ) }
-					className="edit-site-patterns__dropdown"
-					popoverProps={ { placement: 'bottom-end' } }
-					toggleProps={ {
-						className: 'edit-site-patterns__button',
-						describedBy: sprintf(
-							/* translators: %s: pattern name */
-							__( 'Action menu for %s pattern' ),
-							item.title
-						),
-					} }
-				>
-					{ ( { onClose } ) => (
-						<MenuGroup>
-							{ isCustomPattern && ! hasThemeFile && (
-								<RenameMenuItem
-									item={ item }
-									onClose={ onClose }
-								/>
-							) }
-							<DuplicateMenuItem
-								categoryId={ categoryId }
-								item={ item }
-								onClose={ onClose }
-								label={ __( 'Duplicate' ) }
-							/>
-							{ item.type === PATTERN_TYPES.user && (
-								<MenuItem onClick={ () => exportAsJSON() }>
-									{ __( 'Export as JSON' ) }
-								</MenuItem>
-							) }
-
-							{ isCustomPattern && (
-								<MenuItem
-									isDestructive={ ! hasThemeFile }
-									onClick={ () =>
-										setIsDeleteDialogOpen( true )
-									}
-								>
-									{ hasThemeFile
-										? __( 'Clear customizations' )
-										: __( 'Delete' ) }
-								</MenuItem>
-							) }
-						</MenuGroup>
-					) }
-				</DropdownMenu>
+				<PatternActions item={ item } />
```

Comment:
```suggestion
				<PatternActions
					toggleProps={ { className: 'edit-site-patterns__button' } }
					item={ item }
				/>
```

It seems that the pattern action button is no longer visible on the pattern page. To apply [this style](https://github.com/WordPress/gutenberg/blob/2642697ff65f6f011044f8939f9f7ed70af2945c/packages/edit-site/src/components/page-patterns/style.scss#L166-L168), give the toggle button a class name.

![pattern-dropdown-button](https://github.com/WordPress/gutenberg/assets/54422211/2ef3f14f-9193-4b3a-a458-97d99c7fab5c)

Reply:
Oh good spotting. Thanks 👍🏻